---
title: Bioinformatics
---

# [Bioinformatics](https://en.wikipedia.org/wiki/Bioinformatics)

Personally quite excited about developments of [Foregen](https://www.foregen.org/) & am donating to the cause. Regenerating tissue/organs fully is incredibly exciting.

## Links

- [Rosalind](http://rosalind.info/problems/locations/) - Platform for learning bioinformatics and programming through problem solving.
- [Awesome Bioinformatics](https://github.com/danielecook/Awesome-Bioinformatics)
- [STUDY WITH ME | Computational Biology (2019)](https://www.youtube.com/watch?v=DoTBob1_IvI)
- [Primer on statistical mechanics for biochemists](https://github.com/jchodera/statmech-for-biochemists)
- [Kindred](https://github.com/jakelever/kindred) - Python biomedical relation extraction package that uses a supervised approach (i.e. needs training data).
- [fastq-rs](https://github.com/aseyboldt/fastq-rs) - Can process fastq files at about the speed of the coreutils.
- [nglview](https://github.com/arose/nglview) - Jupyter widget to interactively view molecular structures and trajectories.
- [ngl](https://github.com/arose/ngl) - WebGL protein viewer.
- [Mark Zuckerberg Live With Joe DeRisi & Steve Quake || The Future of Technology & Society (2019)](https://www.youtube.com/watch?v=vdUpF2dmnc8)
- [The Biostar Handbook](https://www.biostarhandbook.com/) - Introduces readers to bioinformatics, the scientific discipline at the intersection of biology, computer science, and statistical data analytics dedicated to the digital processing of genomic information.
- [Michael Levin | 2017 Allen Frontiers Symposium](https://www.youtube.com/watch?v=uAEJ0Q2uiNM)
- [Lattice](https://latticeautomation.com/) - Creates design-automation workflows that fundamentally change engineering biology. ([GitHub](https://github.com/Lattice-Automation))
- [GuacaMol](https://github.com/BenevolentAI/guacamol) - Benchmarks for generative chemistry.
- [Single-cell RNA-seq pseudotime estimation algorithms](https://github.com/agitter/single-cell-pseudotime)
- [Allen Institute for Brain Science Toolkit](https://portal.brain-map.org/explore/toolkit)
- [Computer-Designed Organisms](https://cdorgs.github.io/) - Scalable pipeline for creating functional novel lifeforms.
- [BioGrakn](https://github.com/graknlabs/biograkn) - Provides an intuitive way to query interconnected and heterogeneous biomedical data in one single place. ([Article](https://blog.grakn.ai/biograkn-accelerating-biomedical-knowledge-discovery-with-a-grakn-knowledge-graph-84706768d7d4))
- [AlphaFold: Using AI for scientific discovery (2020)](https://deepmind.com/blog/article/AlphaFold-Using-AI-for-scientific-discovery) ([HN](https://news.ycombinator.com/item?id=22087788))
- [Rust-Bio](https://github.com/rust-bio/rust-bio) - Provides implementations of many algorithms and data structures that are useful for bioinformatics.
- [Low-N protein engineering with data-efficient deep learning (2020)](https://www.biorxiv.org/content/10.1101/2020.01.23.917682v1)
- [Machine Boss](https://github.com/evoldoers/machineboss) - Bioinformatics Open Source Sequence machine.
- [atomium](https://github.com/samirelanduk/atomium) - Python molecular modeller (with .pdb/.cif/.mmtf parsing and production).
- [PoincareMaps](https://github.com/facebookresearch/PoincareMaps) - Poincare maps recover continuous hierarchies in single-cell data.
- [OpenMM](https://github.com/openmm/openmm) - Toolkit for molecular simulation using high performance GPU code. ([OpenMM Cookbook](https://openmm.github.io/openmm-cookbook/dev/index.html))
- [Guide to help wet lab biologists learn computational biology (2020)](https://www.reddit.com/r/bioinformatics/comments/fiwtwx/working_from_home_i_made_a_guide_to_help_wet_lab/)
- [BiGG Models](http://bigg.ucsd.edu/) - Search the database by model, reaction, metabolite, or gene. ([Code](https://github.com/sbrg/bigg_models))
- [Systems Biology and Biotechnology Specialization course](https://www.coursera.org/specializations/systems-biology)
- [Path to a free self-taught education in Bioinformatics](https://github.com/ossu/bioinformatics)
- [How to Build a Biotech](https://www.celinehh.com/how-to-build-a-biotech) ([HN](https://news.ycombinator.com/item?id=23816390))
- [Unified rational protein engineering with sequence-based deep representation learning (2019)](https://www.nature.com/articles/s41592-019-0598-1)
- [Deep Molecular Programming: A Natural Implementation of Binary-Weight ReLU Neural Networks (2020)](https://arxiv.org/abs/2003.13720) ([HN](https://news.ycombinator.com/item?id=22768143))
- [Computationally Comparing Biological Networks and Reconstructing Their Evolution (2012)](http://www.robpatro.com/newsite/documents/dissertation.pdf)
- [Bioinformatics Specialization courses](https://www.coursera.org/specializations/bioinformatics)
- [Scanpy](https://github.com/theislab/scanpy) - Single-Cell Analysis in Python.
- [elPrep](https://github.com/ExaScience/elprep) - High-performance tool for preparing .sam/.bam files for variant calling in sequencing pipelines.
- [Announcing Sylph](https://sylph.io/blog/announce.html)
- [Interactive bioimage analysis with Python and Jupyter (2020)](https://www.youtube.com/watch?v=Y3pB3wnOivE)
- [Biomedical Applications of Electrical Stimulation (2020)](https://pubmed.ncbi.nlm.nih.gov/31974658/)
- [a16z invested bio companies](https://twitter.com/vijaypande/status/1265727854836211713)
- [MultiQC](https://multiqc.info/) - Searches a given directory for analysis logs and compiles a HTML report. ([Code](https://github.com/ewels/MultiQC))
- [Diyhplus Wiki](https://diyhpl.us/wiki/) - Wiki for open source hardware, do-it-yourself biohacking and practical transhumanism. ([Code](https://github.com/kanzure/diyhpluswiki))
- [Tellurium](https://github.com/sys-bio/tellurium) - Python environment for reproducible dynamical modeling of biological networks. ([Web](http://tellurium.analogmachine.org/))
- [Chemlambda](https://chorasimilarity.github.io/chemlambda-gui/index.html) - Graph rewriting system derived from graphic lambda calculus [1] which can be seen as a simple model of chemical or biological computing. ([Code](https://github.com/chorasimilarity/chemlambda-gui))
- [FPbase](https://www.fpbase.org/) - Fluorescent Protein Database. ([HN](https://news.ycombinator.com/item?id=23569427))
- [Cell Biology by the Numbers](http://book.bionumbers.org/)
- [CReM](https://github.com/DrrDom/crem) - Open-source Python framework to generate chemical structures using a fragment-based approach.
- [QSAR modeling software and virtual screening](http://qsar4u.com/)
- [Axial - Founder-driven life sciences companies (2020)](https://axial.substack.com/p/axial-founder-driven-life-sciences)
- [Manolis Kellis research](http://web.mit.edu/manoli/)
- [Pumas AI](https://pumas.ai/) - Platform for pharmaceutical modeling and simulation. ([HN](https://news.ycombinator.com/item?id=24126754))
- [Mitochondrial dynamics in postmitotic cells regulate neurogenesis (2020)](https://science.sciencemag.org/content/369/6505/858.full) ([Tweet](https://twitter.com/VanderhaeghenP2/status/1294168825735319553))
- [PVDH Lab](https://pvdhlab.org/) - Stem cell and Developmental neurobiology.
- [Applying tech frameworks to biotech: key differences](https://www.celinehh.com/tech-vs-biotech) ([HN](https://news.ycombinator.com/item?id=24262336))
- [Europe’s biotech renaissance (2020)](https://www.nature.com/articles/s41587-020-0483-6)
- [Awesome DeepBio](https://github.com/gokceneraslan/awesome-deepbio)
- [Coming up with ideas for biotech startups (2020)](https://www.youtube.com/watch?v=MTA5LajqdIc)
- [Analyzing toehold sequences for synthetic biology (2020)](https://news.harvard.edu/gazette/story/2020/10/analyzing-toehold-sequences-for-synthetic-biology/)
- [R, Data Science, & Computational Biology (2020)](https://changelog.com/practicalai/107)
- [Bioinformatics Algorithms book](https://www.bioinformaticsalgorithms.org/read-the-book)
- [ASAP](https://github.com/BingqingCheng/ASAP) - Automatic Selection And Prediction tools for materials and molecules.
- [The second decade of synthetic biology: 2010–2020](https://www.nature.com/articles/s41467-020-19092-2) ([HN](https://news.ycombinator.com/item?id=24786151))
- [BioDesign Research Conference](https://www.biodesign-conference.com/index.php)
- [Cryo–electron microscopy breaks the atomic resolution barrier at last (2020)](https://www.sciencemag.org/news/2020/10/cryo-electron-microscopy-breaks-atomic-resolution-barrier-last) ([HN](https://news.ycombinator.com/item?id=24869727))
- [Deep Learning for Graphs in Chemistry and Biology](https://github.com/mufeili/DL4MolecularGraph)
- [Amorphous computing](https://en.wikipedia.org/wiki/Amorphous_computing) - Refers to computational systems that use very large numbers of identical, parallel processors each having limited computational ability and local interactions.
- [Awesome Single Cell](https://github.com/seandavi/awesome-single-cell) - Community-curated list of software packages and data resources for single-cell, including RNA-seq, ATAC-seq, etc.
- [MacLean Lab](http://macleanlab.usc.edu/) - Stem Cell Systems Biology Research.
- [Cahan Lab](http://www.cahanlab.org/) - Stem cell engineering, developmental biology, and cancer biology research.
- [Tufts Uni: Levin Biology Lab](https://ase.tufts.edu/biology/labs/levin/Default.htm) - Investigating information storage and processing in biological systems.
- [Nextflow](https://github.com/nextflow-io/nextflow) - Bioinformatics workflow manager that enables the development of portable and reproducible workflows. ([Web](https://nf-co.re/)) ([GitHub](https://github.com/nf-core))
- [Quantum deep field for molecule](https://github.com/masashitsubaki/QuantumDeepField_molecule)
- [AlQuraishi Lab at Columbia University](https://www.aqlab.io/) - Machine Learning, Molecules, Systems Biology research. ([GitHub](https://github.com/aqlaboratory))
- [AlphaFold: a solution to a 50-year-old grand challenge in biology, protein folding (2020)](https://deepmind.com/blog/article/alphafold-a-solution-to-a-50-year-old-grand-challenge-in-biology) ([HN](https://news.ycombinator.com/item?id=25253488)) ([Explained](https://twitter.com/AdamRutherford/status/1333449928316743683)) ([Lex explains](https://www.youtube.com/watch?v=W7wJDJ56c88))
- [AlphaFold2 @ CASP14: “It feels like one’s child has left home.” (2020)](https://moalquraishi.wordpress.com/2020/12/08/alphafold2-casp14-it-feels-like-ones-child-has-left-home/) ([Tweet](https://twitter.com/MoAlQuraishi/status/1336351187369603088)) ([HN](https://news.ycombinator.com/item?id=25396736))
- [Alphafold2 in PyTorch](https://github.com/lucidrains/alphafold2)
- [ProSPr: Protein Structure Prediction](https://github.com/dellacortelab/prospr)
- [MiniFold](https://github.com/EricAlcaide/MiniFold) - Deep Learning for Protein Structure Prediction inspired by DeepMind AlphaFold algorithm.
- [What is protein folding? A brief explanation (2020)](https://rootsofprogress.org/alphafold-protein-folding-explainer) ([HN](https://news.ycombinator.com/item?id=25261591))
- [HH-suite3 for sensitive sequence searching](https://github.com/soedinglab/hh-suite) - Software package for sensitive protein sequence searching based on the pairwise alignment of hidden Markov models (HMMs).
- [Soding Lab](https://www.mpibpc.mpg.de/soeding) - Quantitative and Computational Biology Research. ([GitHub](https://github.com/soedinglab))
- [Protein folding explained (2020)](https://www.youtube.com/watch?v=KpedmJdrTpY)
- [sfaira](https://github.com/theislab/sfaira) - Data and model repository for single-cell data.
- [Theis Lab](https://www.helmholtz-muenchen.de/icb/) - Institute of Computational Biology. ([GitHub](https://github.com/theislab))
- [PostEra](https://postera.ai/) - Medicinal Chemistry Powered by Machine Learning.
- [Bioregistry](https://github.com/bioregistry/bioregistry) - Integrative meta-registry of biological databases, ontologies, and nomenclatures.
- [Nature Biotechnology](https://www.nature.com/nbt/)
- [PyMOL](https://pymol.org/) - Molecular visualization system. ([Code](https://github.com/schrodinger/pymol-open-source))
- [Bioinformatics Chat](https://bioinformatics.chat/) - Podcast about computational biology, bioinformatics, and next generation sequencing.
- [OpenCADD](https://github.com/volkamerlab/opencadd) - Python library for structural cheminformatics.
- [Ask HN: Best way to learn computational biology/immunology? (2021)](https://news.ycombinator.com/item?id=25801821)
- [Lumol](https://github.com/lumol-org/lumol) - Classical molecular simulation engine that provides a solid base for developing new algorithms and methods. ([Web](https://lumol.org/))
- [Tasks Assessing Protein Embeddings (TAPE)](https://github.com/songlab-cal/tape) - Set of five biologically relevant semi-supervised learning tasks spread across different domains of protein biology.
- [Center for Computational Biology](https://ccb.berkeley.edu/)
- [Yun S. Song Research](https://people.eecs.berkeley.edu/~yss/) ([Song Lab](https://people.eecs.berkeley.edu/~yss/group.html)) ([GitHub](https://github.com/songlab-cal))
- [MolStar](https://github.com/molstar/molstar) - Comprehensive macromolecular library. ([Web](https://molstar.org/))
- [How to Start a Biotech Company on a Budget (2021)](https://blog.ycombinator.com/how-to-start-a-biotech-company-on-a-budget/)
- [BioGraPy](https://github.com/apierleoni/BioGraPy) - Biological Graphic tool in Python.
- [FPbase](https://www.fpbase.org/) - The Fluorescent Protein Database. ([Code](https://github.com/tlambert03/FPbase))
- [Cell Ontology](http://obofoundry.org/ontology/cl) - Structured controlled vocabulary for cell types in animals. ([Code](https://github.com/obophenotype/cell-ontology))
- [Squidpy](https://github.com/theislab/squidpy) - Spatial Molecular Data Analysis in Python.
- [TorchProteinLibrary](https://github.com/lupoglaz/TorchProteinLibrary) - PyTorch library of layers acting on protein representations.
- [The Wilke Lab](https://wilkelab.org/) - Computational Evolutionary Biology.
- [parasail](https://github.com/jeffdaily/parasail) - Pairwise Sequence Alignment Library.
- [CellChat](https://github.com/sqjin/CellChat) - R toolkit for inference, visualization and analysis of cell-cell communication from single-cell data.
- [BiLSTM-CNN-CRF architecture for sequence tagging](https://github.com/UKPLab/emnlp2017-bilstm-cnn-crf)
- [scArches](https://github.com/theislab/scarches) - Package to integrate newly produced single-cell datasets into integrated reference atlases.
- [Biotech for the Biocurious (2021)](https://minutes.substack.com/p/biotech-for-the-biocurious)
- [The Century of Biology Newsletter](https://centuryofbio.substack.com/)
- [Papers on machine learning for proteins](https://github.com/yangkky/Machine-learning-for-proteins)
- [Finding a "killer application" for novel biotech: AveXis case study (2020)](https://www.youtube.com/watch?v=n5QyDGhJQSI) ([Tweet](https://twitter.com/paraschopra/status/1381947695980453889))
- [BioModels](https://www.ebi.ac.uk/biomodels/) - Vast repository of mathematical models of biological and biomedical systems.
- [noodles](https://github.com/zaeleus/noodles) - Bioinformatics I/O libraries in Rust.
- [Datamol](https://github.com/datamol-org/datamol) - Molecular Manipulation Made Easy.
- [Role of Bioelectricity During Cell Proliferation in Different Cell Types (2020)](https://pubmed.ncbi.nlm.nih.gov/32714900/)
- [tidysq](https://github.com/BioGenies/tidysq) - Contains tools for analysis and manipulation of biological sequences (including amino acid and nucleic acid – e.g. RNA, DNA – sequences).
- [SeqKit](https://github.com/shenwei356/seqkit) - Cross-platform and ultrafast toolkit for FASTA/Q file manipulation in Go.
- [TaxonKit](https://github.com/shenwei356/taxonkit) - Practical and Efficient NCBI Taxonomy Toolkit.
- [unikmer](https://github.com/shenwei356/unikmer) - Toolkit for nucleic acid k-mer analysis, including set operations on k-mers (sketch) optional with TaxIDs but without count information.
- [ProteinSolver](https://github.com/ostrokach/proteinsolver) - Graph neural network capable of generating novel amino acid sequences that fold into proteins with predetermined topologies. ([Docs](https://docs.proteinsolver.org/))
- [Learning from Protein Structure with Geometric Vector Perceptrons (2020)](https://openreview.net/forum?id=1YLJDvSx6J4) ([Code](https://github.com/drorlab/gvp))
- [Dabble](https://github.com/Eigenstate/dabble) - Membrane protein builder and parameterizer.
- [ffq](https://github.com/pachterlab/ffq) - Fetch run information from the European Nucleotide Archive (ENA). ([Tweet](https://twitter.com/sinabooeshaghi/status/1396920733918322689))
- [MolPAL](https://github.com/coleygroup/molpal) - Molecular Pool-based Active Learning.
- [Delocalization-Induced Molecular Equality (2021)](https://depth-first.com/articles/2021/06/17/delocalization-induced-molecular-equality/)
- [Molecular Assembly Index (2021)](https://depth-first.com/articles/2021/06/04/molecular-assembly-index/)
- [Depth-First Blog](https://depth-first.com/)
- [ChemCore](https://github.com/rapodaca/chemcore) - Cheminformatics toolkit for Rust. ([Article](https://depth-first.com/articles/2020/06/01/chemcore-a-cheminformatics-toolkit-for-rust/))
- [Biochemical Pathway Maps](http://biochemical-pathways.com/#/map/1) ([HN](https://news.ycombinator.com/item?id=27544894))
- [Awesome Biomedical Information Extraction](https://github.com/caufieldjh/awesome-bioie)
- [Biological structure and function emerge from scaling unsupervised learning to 250 million protein sequences (2020)](https://www.biorxiv.org/content/10.1101/622803v4) ([Code](https://github.com/facebookresearch/esm))
- [Highly accurate protein structure prediction with AlphaFold (2021)](https://www.nature.com/articles/s41586-021-03819-2) ([Code](https://github.com/deepmind/alphafold)) ([Tweet](https://twitter.com/demishassabis/status/1415736975395631111)) ([HN](https://news.ycombinator.com/item?id=27848186))
- [AlphaFold 2 is here: what’s behind the structure prediction miracle (2021)](https://www.blopig.com/blog/2021/07/alphafold-2-is-here-whats-behind-the-structure-prediction-miracle/) ([HN](https://news.ycombinator.com/item?id=27891532))
- [AlphaFold Protein Structure Database](https://alphafold.ebi.ac.uk/) ([HN](https://news.ycombinator.com/item?id=27919859))
- [The AlphaFold2 Method Paper: A Fount of Good Ideas (2021)](https://moalquraishi.wordpress.com/2021/07/25/the-alphafold2-method-paper-a-fount-of-good-ideas/) ([Tweet](https://twitter.com/MoAlQuraishi/status/1419355930844618753))
- [AlphaFold Colab](https://colab.research.google.com/github/deepmind/alphafold/blob/main/notebooks/AlphaFold.ipynb)
- [More Protein Folding Progress – What’s It Mean? (2021)](https://blogs.sciencemag.org/pipeline/archives/2021/07/23/more-protein-folding-progress-whats-it-mean) ([HN](https://news.ycombinator.com/item?id=27962246))
- [smof](https://github.com/incertae-sedis/smof) - UNIX-style FASTA tools.
- [AlphaFold2: Are attention and symmetries all you need? (2021)](https://journals.iucr.org/d/issues/2021/08/00/rr5212/rr5212.pdf)
- [ColabFold](https://github.com/sokrypton/ColabFold) - Making Protein folding accessible to all via Google Colab.
- [BioLink API](https://github.com/biolink/biolink-api) - API for linked biological knowledge.
- [AlpahFold, a tentative review (2021)](https://cerebralab.com/AlpahFold,_a_tentative_review)
- [htsget-rs](https://github.com/umccr/htsget-rs) - Bioinformatic file formats accessible to the web.
- [Synthace](https://www.synthace.com/) - Enabling life science, the way it should be done.
- [scispacy](https://github.com/allenai/scispacy) - Full spaCy pipeline and models for scientific/biomedical documents. ([Web](https://allenai.github.io/scispacy/))
- [Bioinformatics Format Specimens](https://github.com/BioJulia/BioFmtSpecimens) - Collection of bioinformatics file format specimens to test against.
- [Single-sequence protein structure prediction using language models from deep learning (2021)](https://www.biorxiv.org/content/10.1101/2021.08.02.454840v1) ([Code](https://github.com/hypnopump/rgn2-replica))
- [Robust, Universal Tree Balance Indices (2021)](https://www.biorxiv.org/content/10.1101/2021.08.25.457695v1) ([Tweet](https://twitter.com/robjohnnoble/status/1432656976165617664))
- [Julia for Biologists (2021)](https://arxiv.org/abs/2109.09973)
- [Hierarchical Generation of Molecular Graphs using Structural Motifs (2020)](https://arxiv.org/abs/2002.03230) ([Code](https://github.com/wengong-jin/hgraph2graph))
- [Broad Institute](https://www.broadinstitute.org/) - Unique, collaborative community pioneering a new model of biomedical science. ([Twitter](https://twitter.com/broadinstitute)) ([GitHub](https://github.com/broadinstitute))
- [INDRA (Integrated Network and Dynamical Reasoning Assembler)](https://github.com/sorgerlab/indra) - Automated model assembly system, originally developed for molecular systems biology and then generalized to other domains. ([Web](http://www.indra.bio/))
- [Reproducible, scalable, and shareable analysis pipelines with bioinformatics workflow managers](https://github.com/GoekeLab/bioinformatics-workflows)
- [Neural Distance Embeddings for Biological Sequences (2021)](https://arxiv.org/abs/2109.09740) ([Code](https://github.com/gcorso/NeuroSEED))
- [Protein complex prediction with AlphaFold-Multimer (2021)](https://www.biorxiv.org/content/10.1101/2021.10.04.463034v1) ([Tweet](https://twitter.com/DeepMind/status/1445335773440585729)) ([HN](https://news.ycombinator.com/item?id=28781518))
- [A cryptography game-changer for biomedical research at scale (2021)](https://www.nature.com/articles/s41467-021-25972-y) ([HN](https://news.ycombinator.com/item?id=28826376))
- [MoleculeX](https://github.com/divelab/MoleculeX) - New and rapidly growing suite of machine learning methods and software tools for molecule exploration.
- [Biomappings](https://github.com/biopragmatics/biomappings) - Community curated and predicted equivalences and related mappings between named biological entities that are not available from primary sources.
- [PheKnowLator](https://github.com/callahantiff/PheKnowLator) - Heterogeneous Biomedical Knowledge Graphs and Benchmarks Constructed Under Alternative Semantic Models.
- [Computational Reproducibility with BioNix (2021)](https://academic.oup.com/gigascience/article/9/11/giaa121/5987272)
- [Self-Alignment Pretraining for Biomedical Entity Representations (2021)](https://aclanthology.org/2021.naacl-main.334/) ([Code](https://github.com/cambridgeltl/sapbert))
- [Bandage](http://rrwick.github.io/Bandage/) - Bioinformatics Application for Navigating De novo Assembly Graphs Easily. ([Code](https://github.com/rrwick/Bandage))
- [Viv](https://github.com/hms-dbmi/viv) - Multiscale visualization of high-resolution multiplexed bioimaging data on the web.
- [PHATE](https://github.com/KrishnaswamyLab/PHATE) - Visualizing Transitions and Structure for Biological Data Exploration.
- [Artificial intelligence reveals nuclear pore complexity (2021)](https://www.biorxiv.org/content/10.1101/2021.10.26.465776v1) ([Tweet](https://twitter.com/jankosinski/status/1453530872427253765))
- [BioNix](https://github.com/PapenfussLab/bionix) - Functional highly reproducible bioinformatics pipelines.
- [Awesome Bioinformatics Formats](https://github.com/kmhernan/awesome-bioinformatics-formats)
- [BioShape Lab](https://bioshape.ece.ucsb.edu/)
- [Bioinformatics repository with more and newer packages](https://github.com/BioArchLinux/Packages)
- [The History of Microbiology – A Personal Interpretation](https://www.annualreviews.org/doi/full/10.1146/annurev-micro-033020-020648)
- [SeqFu](https://github.com/telatin/seqfu2) - General-purpose program to manipulate and parse information from FASTA/FASTQ files, supporting gzipped input files.
- [Bioinformatics algorithms covered in Cambridge Uni Bio Course](https://github.com/A-F-V/Bioinformatics)
- [block aligner](https://github.com/Daniel-Liu-c0deb0t/block-aligner) - SIMD-accelerated library for computing global and X-drop affine gap sequence alignments using an adaptive block-based algorithm.
- [OpenFold](https://github.com/aqlaboratory/openfold) - Trainable & open-source PyTorch reproduction of AlphaFold 2.
- [HSM (Hierarchical Statistical Mechanical model)](https://github.com/aqlaboratory/hsm) - Biophysical prediction of protein-peptide interactions and signaling networks using machine learning. ([Web](https://proteinpeptide.io/))
- [Foregen and the Science of Regeneration with Tyler Drzod and Eric Cunningham (2021)](https://www.youtube.com/watch?v=JTOoc8ToSjc) ([Links & Transcript](https://brendonmarotta.com/8090/foregen-and-regeneration/))
- [Foregen](https://www.foregen.org/) - Regenerating Foreskins. ([Reddit](https://www.reddit.com/r/Foregen/)) ([Commentarium](https://www.foregen.org/commentarium))
- [DeepMind releases massive protein structure database (2021)](https://www.freethink.com/science/protein-structure)
- [ATOM3D: Tasks on Molecules in Three Dimensions](https://www.atom3d.ai/) ([Code](https://github.com/drorlab/atom3d))
- [3D Infomax improves GNNs for Molecular Property Prediction (2021)](https://arxiv.org/abs/2110.04126) ([Code](https://github.com/HannesStark/3DInfomax))
- [Regenerated crustacean limbs are precise replicas (2021)](https://www.biorxiv.org/content/10.1101/2021.12.13.472338v1) ([Tweet](https://twitter.com/michalis_averof/status/1470816451422470156))
- [Dockerfiles and documentation on tools for public health bioinformatics](https://github.com/StaPH-B/docker-builds)
- [Arc Institute](https://arcinstitute.org/) - New institution for curiosity-driven biomedical science and technology. ([Tweet](https://twitter.com/patrickc/status/1471156220425871360)) ([Twitter](https://twitter.com/arcinstitute))
- [SeqLike](https://github.com/modernatx/seqlike) - Unified biological sequence manipulation in Python.
- [Deep Learning for Molecules and Materials Book](https://dmol.pub/intro.html) ([Code](https://github.com/whitead/dmol-book))
- [ProtTrans](https://github.com/agemagician/ProtTrans) - State of the art pre-trained models for proteins.
- [sandbox.bio](https://sandbox.bio/) - Interactive bioinformatics tutorials.
- [Foldseek Search Server](https://search.foldseek.com/search) - Suite for searching and clustering protein structures. ([Code](https://github.com/steineggerlab/foldseek)) ([Tweet](https://twitter.com/thesteinegger/status/1478050357419048960))
- [AlphaFold 2 & Equivariance (2020)](https://fabianfuchsml.github.io/alphafold2/)
- [AlphaFold-Powered Drug Discovery of a Novel CDK20 Inhibitor (2022)](https://arxiv.org/abs/2201.09647) ([HN](https://news.ycombinator.com/item?id=30069402))
- [Graphein](https://github.com/a-r-j/graphein) - Protein Graph Library.
- [A backbone-centred energy function of neural networks for protein design (2022)](https://www.nature.com/articles/s41586-021-04383-5)
- [EquiBind: Geometric Deep Learning for Drug Binding Structure Prediction (2022)](https://arxiv.org/abs/2202.05146) ([Code](https://github.com/HannesStark/EquiBind))
- [FastFold](https://github.com/hpcaitech/FastFold) - Optimizing Protein Structure Prediction Model Training and Inference on GPU Clusters.
- [Graphormer](https://github.com/microsoft/Graphormer) - Deep learning package that allows researchers and developers to train custom models for molecule modeling tasks.
- [Strobealign](https://github.com/ksahlin/StrobeAlign) - Fast short-read aligner. It achieves the speedup by using a dynamic seed size obtained from syncmer-thinned strobemers.
- [Bioinformatics training materials](https://github.com/sib-swiss/training-collection)
- [3 Key Questions to Think About When Designing Proteins Computationally (2022)](https://www.blopig.com/blog/2022/03/computational-protein-design-key-methodological-considerations/) ([Tweet](https://twitter.com/AlissaHummer/status/1509592894658723841))
- [AIMSim](https://github.com/VlachosGroup/AIMSim) - Tool for visualizing diversity in your molecular data-set using structural fingerprints.
- [ProteInfer](https://github.com/google-research/proteinfer) - Approach for predicting the functional properties of protein sequences using deep neural networks.
- [Learning inverse folding from millions of predicted structures (2022)](https://www.biorxiv.org/content/10.1101/2022.04.10.487779v1) ([Code](https://github.com/facebookresearch/esm/tree/main/examples/inverse_folding)) ([Tweet](https://twitter.com/alexrives/status/1513602065557340162))
- [What's next for AlphaFold and the AI protein-folding revolution (2022)](https://www.nature.com/articles/d41586-022-00997-5)
- [AlphaFill](https://github.com/PDB-REDO/alphafill) - Algorithm based on sequence and structure similarity that “transplants” missing compounds to the AlphaFold models.
- [tiwih](https://github.com/brentp/tiwih) - Simple bioinformatics command-line tools I wished I had.
- [Awesome Bioinformatics Benchmarks](https://github.com/j-andrews7/awesome-bioinformatics-benchmarks)
- [GLUE (Graph-Linked Unified Embedding)](https://github.com/gao-lab/GLUE) - Graph-linked unified embedding for single-cell multi-omics data integration.
- [Learning to Extend Molecular Scaffolds with Structural Motifs (2022)](https://arxiv.org/abs/2103.03864) ([Code](https://github.com/microsoft/molecule-generation))
- [Functional regeneration and repair of tendons using biomimetic scaffolds loaded with recombinant periostin (2021)](https://www.nature.com/articles/s41467-021-21545-1)
- [MUSCLE](https://github.com/rcedgar/muscle) - Widely-used software for making multiple alignments of biological sequences.
- [fastBio](https://github.com/ahoarfrost/fastBio) - Deep learning library for biological sequences. Extension of Fastai and PyTorch.
- [glosim](https://github.com/lab-cosmo/glosim) - Python package to compute similarities between molecules and structures.
- [Bioontologies](https://github.com/biopragmatics/bioontologies) - Unified access to biomedical ontologies.
- [PyBioPAX](https://github.com/indralab/pybiopax) - Python implementation of the BioPAX object model.
- [AIMNet](https://github.com/aiqm/aimnet) - Atoms In Molecules Neural Network Potential.
- [BigBIO: Biomedical Datasets](https://github.com/bigscience-workshop/biomedical) - Tools for curating biomedical training data for large-scale language modeling.
- [molcloud](https://github.com/whitead/molcloud) - Make a bunch of molecules.
- [RITA: a Study on Scaling Up Generative Protein Sequence Models (2022)](https://arxiv.org/abs/2205.05789) ([Code](https://github.com/lightonai/RITA))
- [Awesome Molecular Generation](https://github.com/amorehead/awesome-molecular-generation)
- [Geometric Transformers for Protein Interface Contact Prediction (2022)](https://openreview.net/forum?id=CS4463zx6Hi) ([Code](https://github.com/BioinfoMachineLearning/DeepInteract))
- [DIPS-Plus: The Enhanced Database of Interacting Protein Structures for Interface Prediction (2021)](https://arxiv.org/abs/2106.04362) ([Code](https://github.com/BioinfoMachineLearning/DIPS-Plus))
- [Carina](https://github.com/lazear/carina) - Fast proteomics search engine in 1000 lines of code.
- [Cluster Tools](https://github.com/constantinpape/cluster_tools) - Distributed segmentation for bio-image-analysis.
- [BCALM](https://github.com/GATB/bcalm) - Bioinformatics tool for constructing the compacted de Bruijn graph from sequencing data.
- [pyfaidx](https://github.com/mdshw5/pyfaidx) - Efficient pythonic random access to fasta subsequences.
- [AlphaFold reveals the structure of the protein universe (2022)](https://www.deepmind.com/blog/alphafold-reveals-the-structure-of-the-protein-universe) ([HN](https://news.ycombinator.com/item?id=32262856))
- [K-mer File Format: a standardized and compact disk representation of sets of k-mers (2022)](https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btac528/6651834?login=true) ([Tweet](https://twitter.com/yoann_dufresne/status/1553994994498867201))
- [A Guide to Decentralized Biotech](https://future.com/a-guide-to-decentralized-biotech/)
- [Saez-Rodriguez Group](https://saezlab.org/) - Develop software tools for systems level analysis and mechanistic modeling of molecular and biomedical data. ([GitHub](https://github.com/saezlab))
